modules/javafx.graphics/src/main/native-glass/gtk/GlassApplication.cpp

```diff
@@ -445,6 +445,7 @@ static void process_events(GdkEvent* event, gpointer data)
     }
 
     if (ctx != NULL) {
+
```

Comment:
You can probably remove this unnecessary change as this is only one change in the file.